src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/FindPatientSteps.java

```diff
@@ -0,0 +1,91 @@
+package org.openmrs.contrib.isanteplus.qaframework.automation;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.isanteplus.qaframework.RunTest;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.FindPatientPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.HomePage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.LoginPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.test.TestBase;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class FindPatientSteps extends TestBase {
+	
+	private FindPatientPage findPatientPage;
+	
+	private ClinicianFacingPatientDashboardPage dashboardPage;
+	
+	private LoginPage loginPage;
+	
+	private HomePage homePage;
+	
+	@Before(RunTest.HOOK.FINDPATIENT)
+	public void systemLogin() {
+		homePage = loginPage.goToHomePage();
+		
+	}
+	
+	@After(RunTest.HOOK.FINDPATIENT)
+	public void destroy() {
+		quit();
+	}
+	
+	@Given("User clicks on Find Patient App")
```

Comment:
I think the Right backgound condition should be ,
`Given user Vists home page and clicks clicks on Find Patient App`

Reply:
> Given user Vists home page and clicks clicks on Find Patient App

Thanks @mozzy11 , these are two different scenarios, its as if you are combining the two  , One  **Userr visits HomePage**  and **user click on FindPatientApp**. In this case we are only using one single function HomePage.goToFindPatientApp.

Reply:
Thats right @sherrif10

Reply:
but make the method more readable